R/make.R

```diff
@@ -2552,13 +2606,40 @@ full_citation_graph <- function(...)
 #' @examples
 #'
 #' # This is the Franklin graph:
-#' g1 <- graph_from_lcf(12, c(5, -5), 6)
+#' g1 <- graph_from_lcf(shifts = c(5L, -5L), n = 12L, repeats = 6L)
 #' g2 <- make_graph("Franklin")
 #' isomorphic(g1, g2)
 #' @export
 #' @cdocs igraph_lcf_vector
-graph_from_lcf <- lcf_vector_impl
+graph_from_lcf <- function(
```

Comment:
the argument order changed. how bad is that? we had no test for the function :zany_face: 

Once we agree on the changes I'll remove formatting changes since those will be brought in by #1869 anyway.